utils/instructions/Dual/index.ts

```diff
@@ -171,7 +171,7 @@ export async function getConfigInstruction({
       serializeInstructionToBase64(configInstruction)
     )
 
-    const initStrikeInstruction = await so.createInitStrikeInstruction(
+    const initStrikeInstruction = await so.createInitStrikeReversibleInstruction(
```

Comment:
This will only allow reversibles to be created now from realms correct?

Reply:
Correct. The biggest difference is just updating the sdk to the new version which makes config use configV3 which enables reversible